cli/cmd/inject.go

```diff
@@ -30,9 +30,17 @@ const (
 	udpDesc            = "pod specs do not include UDP ports"
 )
 
+type injectProxy int
```

Comment:
TIOLI: Don't you think that `proxyInjectionMode` or simply `injectionMode` will commnicate intent in a more clear way?

Reply:
Good point. And that way I can revert `resourceTransformerInject.inject` back to `resourceTransformerInject.injectProxy`. I had renamed it to avoid matching the enum name.